src/lua/fiber.c

```diff
@@ -527,6 +542,17 @@ lbox_fiber_new(struct lua_State *L)
 	return 1;
 }
 
+struct fiber *
+get_fiber(struct lua_State *L)
+{
+	if (lua_gettop(L)) {
```

Comment:
In the new code we do not use implicit casts between integers and booleans. If you need to check the stack is not empty, you need to do `lua_gettop(L) != 0`.

Reply:
Fixed